superchain/superchain.go

```diff
@@ -232,6 +232,8 @@ type ContractVersions struct {
 	OptimismMintableERC20Factory string `yaml:"optimism_mintable_erc20_factory"`
 	OptimismPortal               string `yaml:"optimism_portal"`
 	SystemConfig                 string `yaml:"system_config"`
+	// Superchain-wide contracts:
+	ProtocolVersions string `yaml:"protocol_versions"`
```

Comment:
Where does this get populated from? Have we already implemented the glue code to populate this from the `superchain.yaml` files?

Reply:
This is the "declared version", and gets populated from the `semver.yaml` files, leaning on existing code which unmarshals into this struct (and new line in those files I have populated on this PR). 

The corresponding address is already being loaded from the `superchain.yaml` file and accessible from the data exported from the `superchain` Go module. The new test I added "knows" to grab that address to make the rpc request and get the "actual" version.

Does that make sense?

Reply:
sgtm